test_src/sljitTest.c

```diff
@@ -6366,10 +6366,13 @@ static void test64(void)
 	CHECK(compiler);
 	sljit_free_compiler(compiler);
 
+#if !(defined SLJIT_WX_EXECUTABLE_ALLOCATOR && SLJIT_WX_EXECUTABLE_ALLOCATOR)
+	/* WX allocator may not reuse the address. */
```

Comment:
and neither might the other allocators if a new chunk was needed, hence making this test fragile.

FWIW it seems to fail pretty consistently in OpenBSD and can't figure out why it would work with its use of a memory address that has been made invalid by "free" so it might be worth skipping the whole test instead.